src/overlays/actors/ovl_En_Kaizoku/z_en_kaizoku.c

```diff
@@ -65,118 +136,1921 @@ const ActorInit En_Kaizoku_InitVars = {
+
+void EnKaizoku_Init(Actor* thisx, PlayState* play) {
+    EnKaizoku* this = THIS;
+    s32 pad;
+    Player* player = GET_PLAYER(play);
+    EffectBlureInit1 blureInit;
+
+    this->actor.hintId = 0x54;
+    this->actor.targetMode = 3;
+    this->actor.colChkInfo.mass = 80;
+    ActorShape_Init(&this->actor.shape, 0.0f, ActorShadow_DrawFeet, 0.0f);
+    this->unk_2CA = this->actor.world.rot.z;
+    this->actor.colChkInfo.health = 8;
+    this->unk_2D4 = this->actor.params & 0x3F;
+    this->unk_2BC = (this->actor.params >> 6) & 0x7F;
+
+    if (this->unk_2CA >= 2) {
+        this->unk_2CA = 0;
+    } else if (this->unk_2CA == 2) {
+        this->unk_2CA = 0;
+    }
+
+    this->unk_2EC = this->actor.world.rot.z;
+    this->actor.world.rot.z = 0;
+    this->actor.colChkInfo.damageTable = &D_80B8AB3C;
+    SkelAnime_InitFlex(play, &this->skelAnime, &D_0600D828, &D_060058B8, this->jointTable, this->morphTable, 24);
+    Collider_InitAndSetCylinder(play, &this->unk_3D4, &this->actor, &D_80B8AB7C);
+    Collider_InitAndSetQuad(play, &this->unk_420, &this->actor, &D_80B8ABA8);
+    blureInit.p1StartColor[0] = blureInit.p1StartColor[1] = blureInit.p1StartColor[2] = blureInit.p1StartColor[3] =
+        blureInit.p2StartColor[0] = blureInit.p2StartColor[1] = blureInit.p2StartColor[2] = blureInit.p1EndColor[0] =
+            blureInit.p1EndColor[1] = blureInit.p1EndColor[2] = blureInit.p2EndColor[0] = blureInit.p2EndColor[1] =
+                blureInit.p2EndColor[2] = 255;
+    blureInit.p2StartColor[3] = 64;
+    blureInit.p1EndColor[3] = blureInit.p2EndColor[3] = 0;
+    blureInit.elemDuration = 8;
+    blureInit.unkFlag = 0;
+    blureInit.calcMode = 2;
+    Effect_Add(play, &this->unk_3D0, EFFECT_BLURE1, 0, 0, &blureInit);
+    Actor_SetScale(&this->actor, 0.0125f);
+    this->actor.flags |= 0x8000000;
+    this->actor.flags &= ~1;
+    if (this->unk_2BC == 127) {
+        this->unk_2BC = -1;
+    }
+    if (this->unk_2BC >= 0 && Flags_GetSwitch(play, this->unk_2BC)) {
+        Actor_MarkForDeath(&this->actor);
+        return;
+    }
+    this->unk_3D4.dim.radius = 20;
+    this->unk_3D4.dim.height = 65;
+    this->unk_3D4.dim.yShift = 0;
+    this->unk_2D6 = this->actor.cutscene;
+    this->actor.world.pos.y = player->actor.world.pos.y + 160.0f;
+    this->unk_144 = func_80B85EA0;
+    this->actor.flags |= 0x400;
+    func_80B85F48(this);
+}
+
+void EnKaizoku_Destroy(Actor* thisx, PlayState* play) {
+    EnKaizoku* this = THIS;
+    Effect_Destroy(play, this->unk_3D0);
+    Collider_DestroyCylinder(play, &this->unk_3D4);
+    Collider_DestroyQuad(play, &this->unk_420);
+    func_801A2ED8();
+}
+
+s32 func_80B85858(EnKaizoku* this, PlayState* play) {
+    Actor* itemAction;
+    Player* player;
+    s16 sp1E;
+
+    player = GET_PLAYER(play);
+    sp1E = this->unk_2B0;
+    for (itemAction = play->actorCtx.actorLists[ACTORCAT_ITEMACTION].first; itemAction != NULL;
+         itemAction = itemAction->next) {
+        if (1) {}
```

Comment:
```suggestion
        //! FAKE:
        if (1) {}
```
To help keep track of these fake matches for later fixing